README.md

```diff
@@ -4,69 +4,82 @@
+| `lsof`              |`lsof`                               |`lsof`                               |
+| `man`               |`man`                                |`man`                                |
+| `nano`              |`nano`                               |`nano`                               |
+| `netcat`            |`NOT AVAILABLE`                      |`NOT AVAILABLE`                      |
+| `netstat`           |`net-tools`                          |`net-tools`                          |
+| `openssh-client`    |`openssh-clients`                    |`openssh-clients`                    |
+| `podman`            |`podman`                             |`podman`                             |
+| `7z`                |`p7zip-plugins`                      |`p7zip-plugins`                      |
+| `ripgrep`           |`<gh releases>`                      |`<gh releases>`                      |
+| `rsync`             |`rsync`                              |`rsync`                              |
+| `scp`               |`openssh-clients`                    |`openssh-clients`                    |
+| `screen`            |`NOT AVAILABLE`                      |`NOT AVAILABLE`                      |
+| `sed`               |`sed`                                |`sed`                                |
+| `shasum`            |`perl-Digest-SHA`                    |`perl-Digest-SHA`                    |
+| `socat`             |`socat`                              |`socat`                              |
+| `sudo`              |`sudo`                               |`sudo`                               |
+| `ss`                |`NOT AVAILABLE`                      |`NOT AVAILABLE`                      |
+| `ssl-cert`          |`NOT AVAILABLE`                      |`NOT AVAILABLE`                      |
+| `stow`              |`stow`                               |`stow`                               |
+| `tail`              |`<built in>`                         |`<built in>`                         |
+| `tar`               |`tar`                                |`tar`                                |
+| `time`              |`time`                               |`time`                               |
+| `tldr`              |`NOT AVAILABLE (fedora only)`        |`NOT AVAILABLE (fedora only)`        |
+| `tmux`              |`NOT AVAILABLE (fedora only)`        |`NOT AVAILABLE (fedora only)`        |
+| `vim`               |`vim`                                |`vim`                                |
+| `wget`              |`wget`                               |`wget`                               |
+| `zip`               |`zip`                                |`zip`                                |
+| `zsh`               |`NOT AVAILABLE (fedora only)`        |`NOT AVAILABLE (fedora only)`        |
+| **TOTAL SIZE**      | **903MB** (341MB compressed)        | **TODO**   |
```

Comment:
Will this be updated with final values or any additional notes before merging the PR?